src/style.css

```diff
@@ -1 +1,111 @@
-/* add styles here */
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  width: 100%;
+  display: flex;
+}
+
+.form--big {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.form--small {
+  height: 42px;
+}
+
+.form__label {
+  position: relative;
+  width: 100%;
+}
+
+.form__label::after{
+  position: absolute;
+  content:"";
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.form__label--big::after {
+  width: 19px;
+  height: 19px;
+  flex-shrink: 0;
+  top: 25px;
+  left: 26px;
+}
+
+.form__label--small::after {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.form__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #E1E7ED;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)